catalog/cataloger_diff_test.go

```diff
@@ -197,6 +197,68 @@ func TestCataloger_Diff_SameBranch(t *testing.T) {
 	}
 }
 
+// TestCataloger_Diff_SameBranchDiffMergedChanges test changes we merge from parent are found in diff between two commits
+func TestCataloger_Diff_SameBranchDiffMergedChanges(t *testing.T) {
```

Comment:
Please add a test of same file name in master and branch1

Reply:
My next PR will include the content check fix